src/custom.scss

```diff
@@ -16,22 +16,45 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
-h1, h2, h3, h4, h5 {
+h1,
+h2,
+h3,
+h4,
+h5 {
   font-family: $ner-header-font;
 }
 
 $primary: #ef4345;
+$dark: #464646;
 
 // change the theme
 $theme-colors: (
   'ner-red': #ef4345,
   'ner-dark-grey': #292929,
+  'ner-dark-bg': #353434,
+  'ner-light-bg': #ffffff,
   'ner-white': #ffffff
 );
 
 @function theme-color($key) {
   @return map-get($theme-colors, $key);
 }
 
+.dark {
```

Comment:
feels funny for some elements of theme to be stored in the theme object and then some other ones in here...?

Reply:
it is kinda funny but this is what the themeName (now just name) field is referring to. i don't think there's a way to define js values like "dark" and "light" for variants as well as css classes all in one spot if that makes sense, so that's why it's separate but the themes do point to these class names in a sense